apis/docs/v1/remove.md

```
@@ -0,0 +1,446 @@
+    Ne = 1;
+    Ge = 2;
+    Gt = 3;
+    Le = 4;
+    Lt = 5;
+  }
+
+  ```
+
+  - Remove.TimestampRequest
+
+        | field | type | label | description |
```

Markdownlint (0.37.0)

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)